### ISSUE TITLE
fileName() and filePath() have no extensions

### ISSUE THREAD

**Issue by shernshiou** [Marak/faker.js#1153]
Recently I updated to the latest faker, but the filePath does not come with extension now.

```
❯ node
Welcome to Node.js v12.21.0.
Type ".help" for more information.
> const faker = require('faker');
undefined
> faker.system.filePath()
'/etc/periodic/avon_toys_applications_mp2'
> faker.system.fileName()
'payment_reunion_pre'
> faker.system.fileExt()
'spf'
```

```
yarn workspace v1.22.5
yarn list v1.22.5
└─ faker@5.5.1
```

edit to include version

**Comment by Marak** [comment]
Thank you for the report @shernshiou 

A few issues to `System` module were introduced due to https://github.com/Marak/faker.js/pull/985/files

Releasing a patched version momentarily.

**Comment by Marak** [comment]
This should be resolved in `v5.5.2`.

It would be great if someone could improve the test coverage on `System` so that this doesn't happen again.
